components/admin_console/schema_admin_settings.jsx

```diff
@@ -833,6 +838,23 @@ export default class SchemaAdminSettings extends React.Component {
                 }
             }
         );
+
+        const results = [];
+        for (const saveAction of this.saveActions) {
+            results.push(saveAction());
+        }
+
+        // This waits for all the save actions to be complete and then checks to see if any of the values are truthy value to determine
+        // if they failed or not (since they should return either null or '' on a successfull call and some error text on a failed operation)
+        const hasSaveActionError = await Promise.all(results).then((values) => values.some(((value) => value))).then((value) => value);
+
+        const hasError = this.state.serverError || hasSaveActionError;
```

Comment:
We should probably set `this.state.serverError` if the save action had an error

Reply:
As discussed through direct communication, `BrandImageSettings` displays its own error so this is not required.